plugins/modules/zos_job_submit.py

```diff
@@ -510,6 +512,14 @@
 DEFAULT_EBCDIC_CHARSET = "IBM-1047"
 
 
+def get_encoding_from_locale(module):
+    rc, stdout, stderr = module.run_command("locale -c charmap")
+    if rc:
+        return None
+    else:
+        return stdout.strip()
+
+
```

Comment:
There is currently a PR that adds this feature to the `encode` util. It might be a good idea to use that, but not a big deal.

Reply:
Perfect, I will wait to close this until your PR is merged, then will update to use your util!